sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveExternalCatalog.scala

```diff
@@ -1411,6 +1411,7 @@ object HiveExternalCatalog {
 
   private[spark] def isHiveCompatibleDataType(dt: DataType): Boolean = dt match {
     case _: AnsiIntervalType => false
+    case _: TimestampNTZType => false
```

Comment:
We need to make a decision here. Storing both ltz and ntz as hive timestamp is also an option. In fact, the hive timestamp is ntz, so storing ntz as hive timestamp is actually correct.

Reply:
> We need to make a decision here. Storing both ltz and ntz as hive timestamp is also an option. In fact, the hive timestamp is ntz, so storing ntz as hive timestamp is actually correct.

Ah, O.K, I misunderstood. So if we decide to store both ltz and ntz to Hive table, this change is not necessary right?

Reply:
Do we decide to not support nth in Hive?

Reply:
Not yet. We are discussing.
https://github.com/apache/spark/pull/34647#discussion_r754237594